src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './TodoList.scss';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
+interface Props {
+  alltodos: Todo[];
+  setSelectedUserId: (id: number) => void;
+  selectedUserId: number;
+}
+
+export const TodoList: React.FC<Props> = ({
+  alltodos,
+  setSelectedUserId,
+  selectedUserId,
+}) => {
+  const [selectedselect, setSelectedselect] = useState('All');
```

Comment:
```suggestion
  const [selectedSelect, setSelectedSelect] = useState('All');
```
selectedSelect is a strange name